internal/cmd/primaryip/primaryip.go

```diff
@@ -14,19 +15,24 @@ func NewCommand(s state.State) *cobra.Command {
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
 	}
+	cmd.AddGroup(&cobra.Group{ID: "general", Title: "General"})
+	cmd.AddGroup(&cobra.Group{ID: "protection", Title: "Protection"})
+	cmd.AddGroup(&cobra.Group{ID: "assign", Title: "Assign"})
+
 	cmd.AddCommand(
-		ListCmd.CobraCommand(s),
-		DescribeCmd.CobraCommand(s),
-		CreateCmd.CobraCommand(s),
-		UpdateCmd.CobraCommand(s),
-		DeleteCmd.CobraCommand(s),
-		AssignCmd.CobraCommand(s),
-		UnAssignCmd.CobraCommand(s),
-		ChangeDNSCmd.CobraCommand(s),
-		EnableProtectionCmd.CobraCommand(s),
-		DisableProtectionCmd.CobraCommand(s),
-		LabelCmds.AddCobraCommand(s),
-		LabelCmds.RemoveCobraCommand(s),
+		util.WithGroup("general", ListCmd.CobraCommand(s)),
+		util.WithGroup("general", DescribeCmd.CobraCommand(s)),
+		util.WithGroup("general", DeleteCmd.CobraCommand(s)),
+		util.WithGroup("general", UpdateCmd.CobraCommand(s)),
+		util.WithGroup("general", LabelCmds.AddCobraCommand(s)),
+		util.WithGroup("general", LabelCmds.RemoveCobraCommand(s)),
+		util.WithGroup("general", ChangeDNSCmd.CobraCommand(s)),
```

Comment:
This is `set-rdns`, for Load Balancer and Floating IP you added it to the additional commands. Maybe we should change the struct name to be the same for every command?

Reply:
Yes, we should do that to avoid confusion